### ISSUE TITLE
Rember extension feature intrusive

### ISSUE THREAD

**Issue by elsassph** [fdorg/flashdevelop#845]
Rembering an extension should only be presented for unknown extensions, eg. not already registered for an existing language.

It avoids silly things like associating .hx to text syntax when creating a temporary new document and changing its syntax.

**Comment by Gama11** [comment]
How do you change the default syntax for an extension if it has already been added to a syntax then?

**Comment by elsassph** [comment]
Right, it needs something smarter...

Maybe we could have a double menu:

    Syntax
    +   Change default syntax to >
         +   ActionScript
         +   Haxe
         +   ...
         +   Text
    +   ActionScript
    +   Haxe
    +    ...
    +    Text

**Comment by Gama11** [comment]
Yeah, that's what sublime text does, and I really like that UI (mentioned it in #627).

**Comment by elsassph** [comment]
Yeah that would be better. @meychi?

**Comment by Meychi** [comment]
Seems good to me but this should be done for 5.1+.
